Makefile

```diff
@@ -82,9 +82,6 @@ race_condition_binary: build
 race_condition_run: race_condition_binary
 	./prometheus.race $(ARGUMENTS)
 
-run: binary
```

Comment:
The tutorials up on boxever.com reference the run target, can we get this back in?